c2corg_ui/static/js/simplesearch.js

```diff
@@ -238,8 +237,12 @@ app.SimpleSearchController.prototype.createDataset_ = function(type) {
         return '<div class="header" dataset="' + type + '">' + this.gettextCatalog_.getString(type) + '</div>';
       }).bind(this),
       footer: function(doc) {
-        return '<p class="suggestion-more"><a href="/' + type + '/keyword/' + encodeURI(doc['query']) + '" class="green-text" translate>More results</a></p>';
-      },
+        if (!this.associationContext_) { // don't add this if you're typing in an add-association-tool
+          return '<p class="suggestion-more"><a href="/' + type + '?q=' + encodeURI(doc['query']) + '" class="green-text">+ see more results</a></p>';
```

Comment:
Can someone remind me how to have an i18n string in the JS docs extracted to the POT file? I have used ``this.gettextCatalog_.getString('see more results')`` but the make command keeps ignoring it.